0-ad-history-mod-package/simulation/templates/structures/athen/stable.xml

```diff
@@ -3,6 +3,12 @@
   <Identity>
     <Civ>athen</Civ>
     <SpecificName>Hippōn</SpecificName>
+    <History>
+        Because cavalry in Ancient Greece posed a comparably insignificant factor in most battles, for a long time, the Athenians saw no need and made no efforts in raising competitive cavalry themselves. If necessary, they instead relied on allies to provide cavalrymen or hired mercenaries. However, in the 5th century BC, the Athenians finally recognized the value of home-grown cavalry, maybe after having faced the skilled Persian horsemen in the Greco-Persian wars themselves, and started recruiting.
```

Comment:
```suggestion
        Because cavalry in Ancient Greece played an insignificant role in most battles, Athenians, for a long time,  saw no need to create a competitive cavalry. Instead, they relied on allies to provide cavalrymen or hired mercenaries. However, in the 5th century BC, the Athenians finally recognized the value of home-grown cavalry, maybe after having themselves faced the skilled Persian horsemen in the Greco-Persian wars, and started recruiting.

```